engines/groovie/detection.cpp

```diff
@@ -75,233 +76,126 @@ static const GroovieGameDescription gameDescriptions[] = {
 			"t7g", "",
 			AD_ENTRY1s("T7GMac", "acdc4a58dd3f007f65e99b99d78e0bce", 1814029),
 			Common::EN_ANY, Common::kPlatformMacintosh, ADGF_MACRESFORK,
-			GUIO5(GUIO_MIDIADLIB, GUIO_MIDIMT32, GUIO_MIDIGM, GUIO_NOASPECT, GAMEOPTION_T7G_FAST_MOVIE_SPEED)
-		},
-		kGroovieT7G, 0
-	},
-
-#if 0
-	// These entries should now be identical to the first T7G Mac entry after
-	// changing the app to only use the data part of the resource fork. They
-	// are left disabled here as a reference.
-
-	// The 7th Guest Mac English (Aztec single disc)
-	{
-		{
-			"t7g", "",
-			AD_ENTRY1s("T7GMac", "6bdee8d0f9eef6d58d02fcd7deec3fb2", 1830783),
-			Common::EN_ANY, Common::kPlatformMacintosh, ADGF_MACRESFORK,
-			GUIO5(GUIO_MIDIADLIB, GUIO_MIDIMT32, GUIO_MIDIGM, GUIO_NOASPECT, GAMEOPTION_T7G_FAST_MOVIE_SPEED)
+			GUIO6(GUIO_MIDIADLIB, GUIO_MIDIMT32, GUIO_MIDIGM, GUIO_NOASPECT, GUIO_NOSFX, GAMEOPTION_T7G_FAST_MOVIE_SPEED)
 		},
-		kGroovieT7G, 0
+		kGroovieT7G
 	},
 
-	// The 7th Guest Mac English (Aztec bundle, provided by Thefinaleofseem)
-	{
-		{
-			"t7g", "",
-			AD_ENTRY1s("T7GMac", "0d595d4b44ae1814082938d051e5174e", 1830783),
-			Common::EN_ANY, Common::kPlatformMacintosh, ADGF_MACRESFORK,
-			GUIO5(GUIO_MIDIADLIB, GUIO_MIDIMT32, GUIO_MIDIGM, GUIO_NOASPECT, GAMEOPTION_T7G_FAST_MOVIE_SPEED)
-		},
-		kGroovieT7G, 0
-	},
-#endif
-
 	// The 7th Guest DOS Russian (Akella)
 	{
 		{
 			"t7g", "",
-			{
-				{ "script.grv", 0, "d1b8033b40aa67c076039881eccce90d", 16659},
-				{ "intro.gjd", 0, NULL, 31711554},
-				AD_LISTEND
-			},
+			AD_ENTRY2s("script.grv", "d1b8033b40aa67c076039881eccce90d", 16659,
+						"intro.gjd", NULL, 31711554),
```

Comment:
Yep, this is a better idea. I will see if I can add the checksum later.

Reply:
I actually decided not to checksum video files so that it would work with any language, the only reason to need the 2nd file is to ensure you're in the root directory